airbyte-webapp/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "0.40.32",
   "private": true,
   "engines": {
-    "node": "16.18.1"
+    "node": "16.18.1",
+    "pnpm": "7.26.2"
   },
   "scripts": {
     "prepare": "cd .. && husky install airbyte-webapp/.husky",
-    "prestart": "npm run generate-client",
-    "start": "node -r ./scripts/dev-overwrites.js ./node_modules/.bin/vite",
-    "prestart:cloud": "npm run generate-client",
-    "start:cloud": "AB_ENV=${AB_ENV-frontend-dev} node -r ./scripts/environment.js -r ./scripts/dev-overwrites.js ./node_modules/.bin/vite",
-    "prebuild": "npm run generate-client",
+    "prestart": "pnpm run generate-client",
+    "start": "NODE_OPTIONS='-r ./scripts/dev-overwrites.js' vite",
```

Comment:
This should be the same as it's currently. This wasn't introduced by `pnpm` but by Vite. They appear multiple times, because Vite forks of workers to run ESLint, TypeScript compiler and Stylelint in, which all run through that same setup code, so you'll see it 4 times. I can take a look (in a different PR), if there's an easy way I can make sure I only log that output if I'm in the "main" process of Vite.

Reply:
I've opened https://github.com/airbytehq/airbyte/pull/22409 which will improve this, as requested by you.